tests/e2e/vars.py

```diff
@@ -0,0 +1,14 @@
+"""
+This file is where usernames, project names, server address, etc can be defined
+These tests should be runnable live on Tableau Online or Tableau Server
+Assuming, of course, that you have a site or system administrator account.
+"""
+
+"""
+DO NOT #### add your credentials to this file.
+You can add them to credentials.py, which will never be checked in.
```

Comment:
I may have missed it, if so I apologize. I cant seem to find [tests/e2e/credentials.py] file under the gitignore file.